build-system/babel-config/test-config.js

```diff
@@ -77,7 +67,6 @@ function getTestConfig() {
   const testPresets = [presetEnv];
   return {
     compact: false,
-    ignore: ignoredModules,
```

Comment:
Let's keep `ignore` here explicitly as null with a comment explaining why. This will help prevent someone accidentally adding it back in the future.

Reply:
Not sure if `ignore: null` will work. From the [docs](https://babeljs.io/docs/en/options#ignore), I believe appears `ignore` must be an array. IIRC, passing in `[]` is equivalent to "ignore nothing".

**More context:** The reason I added `ignore` in the first place is because of `babelify` (but not `babel` or `gulp-babel`). Specifically, if you want any code in `node_modules` to be transformed, you need to set `global: true` and then add `ignore: [...]` to skip the modules you don't want transformed. This option is [supported by `babelify`](https://github.com/babel/babelify#why-arent-files-in-node_modules-being-transformed) but not by `babel` or `gulp-babel` (in the actual config).